peachjam_search/documents.py

```diff
@@ -53,7 +53,7 @@ class SearchableDocument(Document):
     order_outcome = fields.KeywordField(attr="order_outcome.name")
 
     # GenericDocument, LegalInstrument
-    author = fields.KeywordField(attr="author.name")
+    authors = fields.TextField()
```

Comment:
```suggestion
    authors = fields.KeywordField()
```

it must remain a keyword field so we can calculate aggregates. It can safely have multiple values.